ProcessLib/Utils/ProcessUtils.cpp

```diff
@@ -43,17 +44,66 @@ std::vector<std::reference_wrapper<ProcessVariable>> findProcessVariables(
     std::vector<ProcessVariable> const& variables,
     BaseLib::ConfigTree const& pv_config,
     std::initializer_list<std::string>
-        tag_names)
+        tags)
 {
     std::vector<std::reference_wrapper<ProcessVariable>> vars;
-    vars.reserve(tag_names.size());
+    vars.reserve(variables.size());
 
-    for (auto const& tag : tag_names)
+    if (variables.size() > tags.size())
+        DBUG("Found multiple process variables with a same tag.");
+
+    for (auto& tag : tags)
     {
-        vars.emplace_back(findProcessVariable(variables, pv_config, tag));
+        auto vars_per_tag = findProcessVariables(variables, pv_config, tag);
+        vars.insert(vars.end(), vars_per_tag.begin(), vars_per_tag.end());
     }
 
     return vars;
 }
 
+std::vector<std::reference_wrapper<ProcessVariable>> findProcessVariables(
+    std::vector<ProcessVariable> const& variables,
+    BaseLib::ConfigTree const& pv_config,
+    std::string const& tag)
+{
+    std::vector<std::reference_wrapper<ProcessVariable>> vars;
+
+    auto var_names = pv_config.getConfigParameterList<std::string>(tag);
+
+    if (var_names.empty())
+        OGS_FATAL("Config tag <%s> is not found.", tag.c_str());
```

Comment:
Message is wrong. The tag was found, but the result is empty.

Reply:
Rewritten the error message.